src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -31,20 +32,44 @@
   handle-submit)
 
 (defn submit-application [db _]
-  (post "/hakemus/api/application"
-        (create-application-to-submit (:application db) (:form db) "fi")
+  (ajax/post "/hakemus/api/application"
+        (create-application-to-submit (:application db) (:form db) (get-in db [:form :selected-language]))
         :application/handle-submit-response)
   (assoc-in db [:application :submit-status] :submitting))
 
 (register-handler
   :application/submit-form
   submit-application)
 
+(def ^:private lang-pattern #"/(\w{2})$")
```

Comment:
Are we sure this won't match anything else than the languages? Would something like `#"/(?i)(fi|sv|en)$"` be safer?